assignment-2/src/model/model.js

```diff
@@ -45,25 +70,106 @@ class Model extends EventEmitter {
             mm = '0'+mm
         } 
         today = mm + '/' + dd + '/' + yyyy;
-        /////////////////////
+        
+        return today;
+    }
+
+    getAnswersByQuestionId(questionId){
+        var result = new Array(0);
+        var k=0;
+        for(var i=0;i < this.state.answers.length;i++){
+            if(this.state.answers[i].questionid === questionId){
+                result[k++]=(this.state.answers[i]);
```

Comment:
In JS, you can just do `result.push` to add a new element at the end of an array.